packages/services/server/src/index.ts

```diff
@@ -597,6 +605,13 @@ export async function main() {
       return;
     });
 
+    // only for integration testing :)
+    server.post('/cache/personal-access-token-cache/delete/:id', async (req, res) => {
+      await registry.injector.get(PersonalAccessTokensCache).delete({ id: (req.params as any).id });
+      void res.status(200).send({});
+      return;
+    });
```

Comment:
This was the easiest solution to avoid having to do `sleep(1_000 * 60 * 5)` in the test code. I am open for alternative suggestions.